Rakefile

```diff
@@ -28,6 +33,10 @@ namespace :build do
             outfilesuffix: '.html',
             filetype: 'html',
           },
+	  delegate_backend: 'html5',
```

Comment:
Indentation… 😠

Reply:
Fixed. My vim isn't properly setup for ruby still :disappointed: